redaxo/src/core/tests/base/instance_list_pool_trait_test.php

```diff
@@ -11,6 +11,7 @@ class rex_test_instance_list_pool
         getInstanceListPoolKey as public;
     }
 
+    /** @phpstan-ignore-next-line */
```

Comment:
Alternativ das feld auf public stellen (sind ja „nur“ in einem test)?

Reply:
ja gute idee, protected reichte sogar schon.